docs/source-app/glossary/secrets.rst

```diff
@@ -0,0 +1,58 @@
+.. _secrets:
+
+#################
+Encrypted Secrets
+#################
+
+Is your App is using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and decrypted at runtime.
+
+.. tip::
+	For non-sensitive configuration values, we recommend using :ref:`plain-text Environment Variables <environment_variables>`.
+
+***************
+What did we do?
+***************
+
+The ``--secret`` option has been added to the **lightning run app** command. ``--secret`` can be used by itself or alongside ``--env``.
+
+When a Lightning App (App) **runs in the cloud**, the Secret can be exposed to the App using environment variables.
+The value of the secret is encrypted in the Lightning.ai database, and is only decrypted and accessible to
+LightningFlow (Flow) or LightningWork (Work) processes in the cloud (when you use the ``--cloud`` option running your App).
+Lightning allows you to bind Secret values to your App.
+
+----
+
+**********************
+What were we thinking?
+**********************
+
+We understand that many Apps require access to private data like API keys, access tokens, database passwords, or other credentials. And that you need to protect this data.
+We developed this feature to provide you with a secure way to store this data in a way that is accessible to Apps so that they can authenticate third-party services/solutions.
+
+----
+
+*********************
+Use Encrypted Secrets
+*********************
+
+First, a Secret must be created using the admin web UI. Once a Secret is created, you can bind it to any of your Apps. You do not need to create a new Secret for each App if the Secret value is the same.
```

Comment:
I'm lost. 
If you can only create secrets via the UI, why do we have the CLI --secret flag?
This is confusing :0

And if we keep UI as the only option, let's explain the steps as follows:
1. Add a secret to your app (login to lightning.ai, go here and here, add screenshots)
2. Use secret in app code (give code snippet example)

See streamlit docs as a good example: https://docs.streamlit.io/streamlit-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management

Reply:
CRUD for Secrets using the CLI was descoped for this iteration.